WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -369,14 +389,15 @@ private async Task<bool> TryHandleInsufficientBalanceCaseAsync(decimal differenc
 			_info.MinimumRequiredAmount = minimumRequiredAmount;
 
 			var selectPocketsDialog =
-				await NavigateDialogAsync(new PrivacyControlViewModel(_wallet, _info, usedCoins: Transaction?.SpentCoins, isSilent: doSilentPocketSelection));
+				await NavigateDialogAsync(new PrivacyControlViewModel(_wallet, _info, Transaction?.SpentCoins, doSilentPocketSelection));
```

Comment:
Your IDE keeps removing the param names like `usedCoins` and `isSilent:`. Please revert this change and fix your IDE.